tests/withdrawals_test.go

```diff
@@ -0,0 +1,57 @@
+package tests
+
+import (
+	"bytes"
+	"context"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/rlp"
+	"github.com/stretchr/testify/require"
+)
+
+func TestWithdrawalFieldsInBlocks(t *testing.T) {
+	requireBase := require.New(t)
+
+	// start network.
+	net, err := StartIntegrationTestNet(t.TempDir())
+	requireBase.NoErrorf(err, "Failed to start the fake network: ", err)
+	defer net.Stop()
+
+	// run endowment to ensure at least one block exists
+	receipt, err := net.EndowAccount(common.Address{42}, 1)
+	requireBase.NoError(err)
+	requireBase.Equal(receipt.Status, types.ReceiptStatusSuccessful, "failed to endow account")
+
+	// get client and block
+	client, err := net.GetClient()
+	requireBase.NoError(err, "Failed to get the client: ", err)
+	defer client.Close()
+	block, err := client.BlockByNumber(context.Background(), receipt.BlockNumber)
```

Comment:
I understand using one client for the complete test scope. Nevertheless, the block is only used in the scope of the next test case and should be queried inside. 

note: i do not think we need cases neither in this test.

Reply:
sub tests were added simply to distinct what was being checked, since neither affect the context, they could be part of the same test, yes, but I rather leave it as sub test since what is being checked is different